src/test/scala/ai/privado/exporter/SinkProcessingExporterTest.scala

```diff
@@ -0,0 +1,110 @@
+package ai.privado.exporter
+
+import ai.privado.model.{CatLevelOne, Constants, FilterProperty, Language, NodeType, RuleInfo}
+import ai.privado.model.exporter.SinkProcessingModel
+import ai.privado.exporter.DataflowExporterValidator
+import ai.privado.testfixtures.JavaFrontendTestSuite
+import ai.privado.rule.{RuleInfoTestData, SinkRuleTestData}
+import ai.privado.cache.RuleCache
+import io.shiftleft.semanticcpg.language.*
+
+class SinkProcessingExporterTest
+    extends JavaFrontendTestSuite
+    with DataflowExporterValidator
+    with SinkExporterValidator {
+
+  val sinkRule = List(
+    RuleInfo(
+      "Storages.Local",
+      "Local Storage",
+      "",
+      FilterProperty.METHOD_FULL_NAME,
+      Array("local.com"),
+      List("(?i).*(localStorage).*(save|find).*"),
+      false,
+      "",
+      Map(),
+      NodeType.REGULAR,
+      "",
+      CatLevelOne.SINKS,
+      catLevelTwo = Constants.storages,
+      Language.JAVA,
+      Array()
+    )
+  )
+
+  val ruleCache = RuleCache().setRule(
+    RuleInfoTestData.rule
+      .copy(sources = RuleInfoTestData.sourceRule, sinks = sinkRule)
+  )
+
+  "Simple sink processing dataflow Test" should {
+    val cpg = code(
+      """
+        |import Dummy.LocalStorage;
+        |
+        |class Main {
+        |   public void printValues() {
+        |     String firstName = "first";
+        |
+        |     LocalStorage localStorage = new LocalStorage();
+        |
+        |     localStorage.save(firstName);
+        |   }
+        |}
+        |""".stripMargin,
+      "index.java"
+    )
+      .withRuleCache(ruleCache)
+
+    "Sink processing should have correct node" in {
+      val outputJson           = cpg.getPrivadoJson()
+      val processingList       = getSinkProcessings(outputJson).flatMap(_.occurrences)
+      val storageDataflow      = getStorageFlows(outputJson)
+      val dataflowSinkElements = storageDataflow.flatMap(_.sinks).flatMap(_.paths).map(_.path.lastOption.get)
```

Comment:
I believe the size of `dataflowSinkElements` will be `1`. The validation of the changes done in this PR will be when there will be multiple sink processing in output.

Update the code snippet with 
```
public void printValues() {
        |     String firstName = "first";
        |
        |     LocalStorage localStorage1 = new LocalStorage();
        |
        |     localStorage1.save(firstName);

LocalStorage localStorage2 = new LocalStorage();
        |
        |     localStorage2.save("dummyvalue2");
        
        LocalStorage localStorage3 = new LocalStorage();
        |
        |     localStorage3.save("dummyValue3");
        |   }